doc/source/examples_source/00-basic/04-ptrace_pathline.py

```diff
@@ -0,0 +1,125 @@
+
+
+###############################################################################
+# Create a clip plane
+# -------------------
+# Create a clip through the domain, at X = 0.75.
+# We first call up the default clip part, set attributes, and then create the clip
+# MESHPLANE sets the type of clip (e.g. X, Y, Z, R, T, Z, etc)
+# TOOL sets the tool to create the clip from.
+# VALUE is the location of the clip.
+# DOMAIN controls intersection vs inside vs outside etc.
+# Parent Part is named 'Part by All Elements'
+#
+
+clip = eocore.DEFAULTPARTS[session.ensight.PART_CLIP_PLANE]
+parent_parts = eocore.PARTS["Part by All Elements"][0]
+
+attrs = []
+attrs.append(["MESHPLANE", eonums.MESH_SLICE_X])
+attrs.append(["TOOL", eonums.CT_XYZ])
+attrs.append(["VALUE", 0.75])
+attrs.append(["DOMAIN", eonums.CLIP_DOMAIN_INTER])
+clip = clip.createpart(name="X_Clip", sources=[parent_parts], attributes=attrs)
+
+
+###############################################################################
+# Create a Pathline Trace Emitting from the Clip Part
+# ---------------------------------------------------------
+# Using the 2D parts as the parent (model is 2d), with the "from Part" as the emission type
+# "VITESSE" as the vector, and 25 points along the Clip line as emitter locations.
+# We also setup to first emit the Pathlines at time = 4 seconds.
+# and Emit NEW pathlines ever 20 seconds after that. (They will follow NEW path)
+
+# .. image:: /_static/04_pathline_1.png
+
+emitter_part = clip
+parent_parts = eoutil.parts.select_parts_by_dimension(2)
+npts = 25  # number of emitters
+vector_var = varname  # Vector variable to use
+pathline_part = eoutil.parts.create_particle_trace_from_parts(
+    "Pathline",
+    vector_var,
+    parts=emitter_part,
+    num_points=npts,
+    source_parts=parent_parts,
+    pathlines=True,
+    emit_time=4.0,
+    delta_time=20.0,
+)
+session.show("image", width=800, height=600)
+
+
+###############################################################################
+# Change Visual Attributes
+# ----------------------------------------------------------
+#  Modify the attributes of the pathlines and animate over time
+
+# .. image:: /_static/04_pathline_2.png
+
+pathline_part.REPRESENTATION = eonums.TRACE_TUBE
+pathline_part.WIDTHSCALEFACTOR = 0.012
+session.show("image", width=800, height=600)
+
+
+###############################################################################
+# Animate the Pathlines
+# ----------------------------------------------------------
+#  Turn OFF the pathline lines visibility (to see the animate under)
+#  Turn ON the animate pathlines.
+#  Change to Sphere representation, size, and adjust speed and length.
+
+# .. image:: /_static/04_pathline_3.mp4
+
+pathline_part.VISIBLE = False
+pathline_part.ANIMATE = True
+eocore.HEADTYPE = eonums.ATRACE_HEAD_SPHERE
+eocore.HEADSCALE = 0.3
+session.ensight.solution_time.play_forward()
+session.show("image", width=800, height=600)
```

Comment:
This is fine, but a suggestion might be to use:

session.show("animation", width=800, height=600, fps=15)

will generate an mp4 (like your image) and embed it in the notebook cell.  Just trying to expose more aspects of the API.